### PR TITLE
Edits language

### PR DESCRIPTION
**Pull request by emileighoutlaw**
Clarifies paper and plastic passes. Lowers reading level. Trims content.

### CODE REVIEW

File: ekip/everykid/templates/plan-your-trip/index.html [hunk 1]
```diff
@@ -28,14 +27,12 @@
...
-
+      <h2>Explore parks and sites</h2>
       <ul>
-        <li><a href="http://www.fws.gov/refuges/">National Wildlife Refuge System</a>: national wildlife refuges</li>
-        <li><a href="http://www.discovertheforest.org">Discover the Forest:</a>: national forests and other areas</li>
-        <li><a href="http://findyourpark.com/find">Find Your Park</a>: national parks</li>
+        <li>Find places with protected animals on the <a href="http://www.fws.gov/refuges/">National Wildlife Refuge System</a> website.</li>
+        <li>Find forests on the <a href="http://www.discovertheforest.org">Discover the Forest</a> website.</li>
+        <li>Find parks on the <a href="http://findyourpark.com/find">Find Your Park</a> website.</li>
+        <li>Find marine sites at <a href="http://www.recreation.gov/">recreation.gov</a>.</li>
```
Comment (khandelwal) [inline ekip/everykid/templates/plan-your-trip/index.html]: It's not just marine sites on recreation.gov (it's all inclusive) so I don't know how we want to mention that.

Comment (emileighoutlaw) [inline ekip/everykid/templates/plan-your-trip/index.html]: So this is the NOAA link we're trying to sort out with Chris. Should I just omit the marine sites part for now?

Comment (khandelwal) [inline ekip/everykid/templates/plan-your-trip/index.html]: Yes, that might be best.

File: ekip/everykid/templates/plan-your-trip/index.html [hunk 2]
```diff
@@ -28,14 +27,11 @@
...
-      <p>These cool trip planning tools (from our partner agencies), can help you find a destination. There are many federal lands and waters you can visit, and each presents a whole variety of activities you can do.</p>
-
+      <h2>Explore parks and sites</h2>
       <ul>
-        <li><a href="http://www.fws.gov/refuges/">National Wildlife Refuge System</a>: national wildlife refuges</li>
-        <li><a href="http://www.discovertheforest.org">Discover the Forest:</a>: national forests and other areas</li>
-        <li><a href="http://findyourpark.com/find">Find Your Park</a>: national parks</li>
+        <li>Find places with protected animals on the <a href="http://www.fws.gov/refuges/">National Wildlife Refuge System</a> website.</li>
+        <li>Find forests on the <a href="http://www.discovertheforest.org">Discover the Forest</a> website.</li>
+        <li>Find parks on the <a href="http://findyourpark.com/find">Find Your Park</a> website.</li>
```
Comment (emileighoutlaw) [inline ekip/everykid/templates/plan-your-trip/index.html:34]: Sorted, @khandelwal !

**Pull request merged**